corehq/apps/translations/app_translations/utils.py

```diff
@@ -226,3 +246,69 @@ def update(self, rows):
         a function like django.contrib.messages.error, and the second is a string.
         '''
         raise NotImplementedError()
+
+
+# TODO: (2019-04-26) Drop support for legacy names after all translations could possibly have been uploaded
```

Comment:
The todo (start date) seems a little strange. What about changing it to the end date? It might be a bit easier to reason about the delete if we decide an end date now